spec/test/helpers.py

```diff
@@ -78,7 +78,8 @@ def check_spec_test_env():
         # copy the contents of /app/spec into /spec
         shutil.rmtree(_CONF["spec_paths"]["root"], ignore_errors=True)
         shutil.copytree("/app/spec", _CONF["spec_paths"]["root"])
-        download_specs()
+        if do_download_specs:
+            download_specs()
```

Comment:
are there any cases where `do_download_specs` is not `True`?

Reply:
Umm this was left in by accident probably